File: C/Activity39.c
#include <stdio.h>
#define PI 3.14159   /* Constant for pi */

/* Function prototype */
float getArea(float radius);

/* Global variable to store result */
float area;

int main() {
    float radius;

    printf("WELCOME TO NICO'S COMPUTING AREA OF THE CIRCLE PROGRAM\n\n");

    /* Ask user input */
    printf("Input radius: ");
    scanf("%f", &radius);

    /* Call function */
    area = getArea(radius);

    /* Display result with 2 decimal places */
    printf("Output: the area is : %.2f\n", area);

    return 0;
}

/* Function definition */
float getArea(float radius) {
    return PI * radius * radius;   /* Formula: πr^2 */
}

File: Java/Activity39.java
import java.util.Scanner;

public class Activity39 {

    public static float getArea(float radius) {
        return (float) (Math.PI * radius * radius); 
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("WELCOME TO NICO’S COMPUTING AREA OF THE CIRCLE PROGRAM");

        System.out.print("Input radius: ");
        float radius = scanner.nextFloat(); 

        float area = getArea(radius);

        System.out.printf("Output: the area is : %.2f%n", area); 
    }
}
